i18n-unused.config.js

```diff
@@ -10,26 +10,31 @@ const path = require("path");
  * It also ensure that we don't match any other similar function calls (e.g. `format("dddd")`).
  *
  * ## Explanation of the regex
- * - (?<!\w): negative lookbehind to ensure that there is no word character before "t" or "i18nKey"
+ * - (?<!\w): negative lookbehind to ensure that there is no word character before "t", "language" or "i18nKey"
  * - (?: ... ): non-capturing group
  *   - t: for the use of `t("..")` OR `t("..", {key: ".."})`
  *     - \(("[^"]*"): captures a string enclosed in double quotes followed by an opening parenthesis
  *     - (?:,\s*\{[^}]*\})?: optional non-capturing group so that we match the optional interpolation object
  *   - i18nKey=".+"[^\w]: for the usage of `<Trans i18nKey="home_hero_subtitle" t={t}>`
  *
  * */
```

Comment:
Extend comment based on the changes you have made to the translationKeyRegex. It would be really helpful to debug issues in future to make updates 🙌